packages/block-library/src/query-pagination-next/index.php

```diff
@@ -26,7 +27,13 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 	if ( $pagination_arrow ) {
 		$label .= $pagination_arrow;
 	}
-	$content = '';
+
+	$wrapper_attributes = get_block_wrapper_attributes();
```

Comment:
This is already defined on line 23, do we need to call it again?

Reply:
[Removed!](https://github.com/WordPress/gutenberg/pull/36681/commits/b9ab20a95cdf7f9e9529e7d39f546729f7390e5b)